src/defineFunction.js

```diff
@@ -92,23 +93,22 @@ type FunctionDefSpec = {|
     props: FunctionPropSpec,
 
     // The handler is called to handle these functions and their arguments.
-    //
     // The function should return an object with the following keys:
     //   - type: The type of element that this is. This is then used in
     //          buildHTML/buildMathML to determine which function
     //          should be called to build this node into a DOM node
     // Any other data can be added to the object, which will be passed
     // in to the function in buildHTML/buildMathML as `group.value`.
-    handler: ?FunctionHandler,
+    handler: ?FunctionHandler<NODETYPE>,
 
     // This function returns an object representing the DOM structure to be
     // created when rendering the defined LaTeX function.
-    // TODO: Port buildHTML to flow and make the group and return types explicit.
+    // TODO: Change `group` to ParseNode<NODETYPE> and make return type explicit.
     htmlBuilder?: (group: *, options: Options) => *,
```

Comment:
What's preventing us from doing this TODO now?

Reply:
I tried it, but there were type failures. This requires handling more type assertions in the various  `htmlBuilder`s and `mathmlBuilder`s. This is the kind of stuff within `buildHTML` and `buildMathML` as well.

I figured that since this is already a lot and is in a good spot and catches a lot of errors already, I'd do the rest in a separate PR.  ;)

Reply:
This is definitely non-trivial and definitely belongs in a separate PR. There would need to be more refinement of the `ParseNode` types. It's better explained by sharing the next PR with you once it's done. ;)